test/e2e/kubelet.go

```diff
@@ -396,7 +396,7 @@ var _ = framework.KubeDescribe("kubelet", func() {
 	})
 
 	// Test host cleanup when disrupting the volume environment.
-	framework.KubeDescribe("host cleanup with volume mounts [Volume][HostCleanup][Flaky]", func() {
+	framework.KubeDescribe("host cleanup with volume mounts [sig-storage][HostCleanup][Flaky]", func() {
```

Comment:
Why the [storage] tag instead of calling SIgDescribe() ?]
What is the pattern for using one or the other?